MANIFEST.in

```diff
@@ -4,5 +4,6 @@ include README.md
 include MANIFEST.in
 include staramr/databases/exclude/data/genes_to_exclude.tsv
 include staramr/tests/integration/data/*.fsa
```

Comment:
So, this is something that's my fault (I forgot about it), but could you include the line below line here?

```
staramr/tests/integration/data/gene-drug-tables/*.tsv
```

Those files were missing from the package (but were just used for testing, so weren't noticed before this).